assembly/fabric8-ide-gwt-app/src/main/resources/org/eclipse/che/public/keycloak/RhCheKeycloak.js

```diff
@@ -284,19 +321,34 @@ function initAnalytics(writeKey){
                             traits.adobeMarketingCloudVisitorId = localStorage['openshiftio.adobeMarketingCloudVisitorId'];
                         }
                         analytics.identify(user.id, traits);
+                        return true;
+                    } else {
+                        post('/api/fabric8-che-analytics/error', "Invalid user data for user " + keycloak.tokenParsed.sub + "\nResponse body: " + request.responseText);
```

Comment:
I don't know the level of javascript there but usually now there is template literals
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

`"Invalid user data for user " + keycloak.tokenParsed.sub + "\nResponse body: " + request.responseText``

-->

```
`Invalid user data for user ${keycloak.tokenParsed.sub}
Response body: ${request.responseText}`
```

Reply:
seems they are bot supported in 2 browsers (not that much use but...). TBH I wouldn't like to take the risk of introducing a breaking change, even if it's only for some users that have old browsers...